src/components/actions/throttle.ts

```diff
@@ -21,8 +21,8 @@ const THROTTLE_QUEUE_THRESHOLD = 200 // ms
 
 type EventObject<T, U> = { name: string; data: T; action: (data: T) => Promise<U> | U }
 
-export class EventThrottle<T, U> {
-  queue: EventObject<T, U>[]
+export class EventThrottle {
+  queue: EventObject<any, any>[]
```

Comment:
Okay, this has to be a truly legit use of `any`. I wanted to keep the Generic on the "EventObject", to enforce that the input "data" has to be consistent with the "action" function, but I *don't* want to have to explicitly specify a type when creating the "Event Throttle" object, as it should handle any type of Event object.

But, annoyingly, it requires a type in this definition, so it has to be `any`, right?